ecc/bls12381/ff/common.go

```diff
@@ -35,16 +35,23 @@ func setString(in string, order []byte) ([]uint64, error) {
 		return nil, errInputRange
 	}
 	inBytes := inBig.FillBytes(make([]byte, len(order)))
-	return setBytes(inBytes, order)
+	return setBytesBounded(inBytes, order)
 }
 
-func setBytes(in []byte, order []byte) ([]uint64, error) {
+func setBytesBounded(in []byte, order []byte) ([]uint64, error) {
 	if isLessThan(in, order) == 0 {
 		return nil, errInputRange
 	}
 	return conv.BytesBe2Uint64Le(in), nil
 }
 
+func setBytesUnbounded(in []byte, order []byte) []uint64 {
+	inBig := new(big.Int).SetBytes(in)
+	inBig.Mod(inBig, new(big.Int).SetBytes(order))
```

Comment:
Are all scalars non-secret?

Reply:
this function must be marked as non-constant time. Do you have any ideas on how to perform this operation in constant time?

Reply:
What length are we applying it to? I think I only need it for double length ones, in which case a reduction is all that is required.